onnxruntime/core/optimizer/qdq_transformer/selectors_actions/shared/utils.cc

```diff
@@ -95,6 +98,13 @@ void RegisterConvSelector(Selectors& qdq_selectors) {
                                  std::move(selector));
 }
 
+void RegisterConvTransposeSelector(Selectors& qdq_selectors) {
+  /* register selector for conv op */
+  std::unique_ptr<NodeGroupSelector> selector = std::make_unique<ConvNodeGroupSelector>();
```

Comment:
>ConvNodeGroupSelector [](http://example.com/codeflow?start=65&length=21)

nit: add a comment to explain that it shares selector with Conv

Reply:
sure